documentdb_test.go

```diff
@@ -252,6 +252,18 @@ func TestCreateDocument(t *testing.T) {
 	assert.NotEqual(t, doc.Id, "")
 }
 
+func TestCreateDocumentWithAppIdentifier(t *testing.T) {
+	client := &ClientStub{}
+	defaultConfig.WithAppIdentifier("documentdb_test.TestCreateDocumentWithAppIdentifier")
+	c := &DocumentDB{client, defaultConfig}
+	// TODO: test error situation, without id, etc...
```

Comment:
What about this TODO?

Reply:
This is copy+pasted from `TestCreateDocument` which I used as the baseline to add coverage for this option. I'll remove the TODO given that there shouldn't be any special error cases for app identifier, so it's not needed for coverage.